haskell-language-server.cabal

```diff
@@ -80,7 +80,7 @@ library
     , hie-bios >= 0.4
     , hslogger
     , lens
-    , ormolu ^>= 0.0.5.0
+    , ormolu ^>= 0.1.2
```

Comment:
Wasn't sure what to set this to exactly. Not sure if this is correct?

Reply:
The caret operator is special syntax to add a PVP conform version constraint.
IIRC, in PVP in the version a.b.c.d, `a.b` are major versions while `c.d` are minors, so should be compatible.
Therefore, this syntax is identical to this constraint (iirc):
```cabal
  , ormolu >= 0.1.2 && < 0.2
``` 
So, yeah that is correct